src/app/Main/Body/QueryPage/QueryPageServices/TableServices/generateOneTableRowObj.service.ts

```diff
@@ -13,29 +13,45 @@ export function generateOneTableRowObjService(
 ): Record<string, string> {
     const tempStatTable: Record<string, string> = {};
     tempStatTable.name = rowName;
-    columnNameValues.forEach((colName: string | number) => {
-        if (
+    columnNameValues.forEach((colName: string) => {
+        const isolateGroupIncudesRowAndCol: boolean =
             _.includes(uniqIsolateColValues, colName) &&
-            _.includes(uniqIsolateRowValues, rowName)
-        ) {
+            _.includes(uniqIsolateRowValues, rowName);
+
+        if (isolateGroupIncudesRowAndCol) {
+            let breakIsolateCountLoop = false;

```

Comment:
(obsolete due to reject below)
unnecessary, you could return directly, see below

Reply:
wrong, you're right